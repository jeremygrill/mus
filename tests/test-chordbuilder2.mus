Int main(){
	
	Chord c;
	Note x;
	Note y;
	Int i;
	Int r;

	x = (11,11|11);
	y = (22,22|22);

	i = 0;
	while(i < 10){
		c = c * (10+i,10|10) * x;	
		i = i + 1;
	}

	c = x * c;

	printc(c);
	return 0;
}