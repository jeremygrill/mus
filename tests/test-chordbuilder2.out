[(10,10|10)(11,11|11)(11,10|10)(11,11|11)(12,10|10)(11,11|11)(13,10|10)(11,11|11)(14,10|10)(11,11|11)(15,10|10)(11,11|11)(16,10|10)(11,11|11)(17,10|10)(11,11|11)(18,10|10)(11,11|11)(19,10|10)(11,11|11)(11,11|11)]
